kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/filesystem/CSVRecordParser.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2013 Cloudera Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kitesdk.data.spi.filesystem;
```

Comment:
Good question. I know right now we're using these methods in SPI-based components outside of Kite, like the Flume sink. These components are generally-useful though, so I'd like to see some sort of utility library eventually.